Cargo.toml

```diff
@@ -79,7 +82,7 @@ tonic-web-wasm-client = "0.7"
 thiserror = "2.0.3"
 tracing = "0.1.41"
 tracing-subscriber = "0.3.15"
-uniffi = "0.29"
+uniffi = "0.29.2"
```

Comment:
do we need to enforce this patch version?

Reply:
Yes. We have a compilation error on `0.29.0` and `0.29.1`. Minimal versions CI job was not passing.